### PR TITLE
Add provider table

### PR DESCRIPTION
**Pull request by qdrn**
TBC. Maybe there are community RPC that can be added @aoudiamoncef ?

### CODE REVIEW

**Comment by aoudiamoncef** [comment]
I tried to make this clean, but I got some issues in gRPC health builtin service health checks which works only on DNS 👎🏻

The plan is: 

1. Update the list of API providers for testnet 24
2. Update the UptimeKuma status pages (I can host It in my infra, no problem, we'll migrate to an official massa server and DNS if it's a success)
3. Add the list in the new documentation and add a small badge in status column https://github.com/louislam/uptime-kuma/pull/1119


A concret example: https://uptimekuma.maoudia.com/status/massa-testnet

File: docs/build/api/providers.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+id: providers
+sidebar_label: Providers
+---
+
+# RPC Providers
+
+You can use any of the following providers to connect to the Massa blockchain.
+The latency experienced may vary depending on the geographical location of the provider.
+You can use multiple providers to enhance redundancy and for load balancing.
```
Comment (Ben-Rey) [inline docs/build/api/providers.mdx:10]:
```suggestion
To interact with the Massa blockchain, you can select from any of the listed providers. Be aware that the latency you encounter may vary depending on the provider's geographical location. To enhance redundancy and facilitate load balancing, consider using multiple providers.
```

**Review by Ben-Rey — APPROVED** [review]

**Comment by aoudiamoncef** [comment]
Hi @qdrn 

I added a concret example of API providers https://github.com/massalabs/docu-dev/commit/53a6a979079ec582a287f3ea44073fa387e3e20b

**Comment by qdrn** [comment]
Hey @aoudiamoncef !

I think what you did is nice. I changed a bit the layout of the page. Can you rebase on the new layout and add the providers ?

**Comment by aoudiamoncef** [comment]
> Hey @aoudiamoncef !
> 
> I think what you did is nice. I changed a bit the layout of the page. Can you rebase on the new layout and add the providers ?
> 
> You can add me as reviewer once it's done.

I'll update and add the missing entries ASAP

**Comment by aoudiamoncef** [comment]
@qdrn I'll add missing providers + a small tutorial to add Massa behind Traefik ASAP, It's an exercice for my homelab

image

**Comment by qdrn** [comment]
> @qdrn I'll add missing providers + a small tutorial to add Massa behind Traefik ASAP, It's an exercice for my homelab

I think the most relevant thing is to list providers. We can decide which kind of monitoring we want to use/promote in a next phase.

**Comment by aoudiamoncef** [comment]
Yep, I'll add the rest of providers Discord Usernames + IPs addresses without a status. We'll see for the monitoring later. 

As we are in maintenance mode. We'll wait until the launch 🚀 of testnet 24 to merde this pr ?

**Comment by qdrn** [comment]
We can merge it once you add them. The documentation is not public anyway.

**Comment by qdrn** [comment]
> > Any progress @aoudiamoncef ?
> 
> I was thinking that sharing IP addresses directly is not a good idea.
> 
> We should use a DNS to point to every IP:PORT
> 
> api-provider-xxxx.massa.net/grpc or api-providers.massa.net/xxxxx/grpc

Question is: are there providers other than altailabs and us that provide endpoints with a DNS ? If not, then let's merge as is.

**Comment by aoudiamoncef** [comment]
We have two choices: use a convention for different available features or add a column to describe the endpoints for JsonRPC or/and gRPC. Ex test.massa.net/api/v1 and grpc://test.massa.net/grpc/api/v1.

May be we should split gRPC and JsonRPC API provider lists ?

Currently, we don't have the information about how to call gRPC. I got a lot issues when I tried to use the raw API providers list in my monitoring node.

**Comment by qdrn** [comment]
Let's add what we have for now in the right format and then we'll add providers as they come.

**Pull request merged**